src/ReadLog.jl

```diff
@@ -0,0 +1,134 @@
+# Read LCM log files directly
+
+# export
+#   LCMlog,
+#   close,
+#   readNextEvent,
+#   handle,
+#   subscribe
+
+# also see JuliaLang's RawFD type, but little information found.
+# www.c4learn/c-programming/c-file-structure-and-file-pointer
+mutable struct FILE
+  level::Cshort
+  token::Cshort
+  bsize::Cshort
+  fd::Cuchar
+  flags::Cuint  # (unsigned flags ;)
+  hold::Cuchar
+  buffer::Ptr{Cuchar}
+  curp::Ptr{Cuchar}
+  istemp::Cuint # (unsigned is temp)
+end
+
+mutable struct lcm_eventlog_t
+  f::Ptr{FILE}
+  eventcount::Clonglong
+end
+const _LCMlog = Ptr{lcm_eventlog_t}
+
+
+struct lcm_eventlog_event_t
+  eventnum::Clonglong
```

Comment:
Likewise, can you please replace the `long long`s and `long`s here with the explicit `Int64` and `Int32` from `eventlog.h`?

Reply:
yes